.github/workflows/tests.yml

```diff
@@ -22,12 +22,29 @@ jobs:
           - "7.3"
           - "7.4"
 
+    env:
+      MYSQL_USER: "travis"
```

Comment:
I guess it's for BC with travis, still, and the user/pwd is hardcoded somewhere in the config files. But maybe it's time to change it? (e.g. to default root/root) `travis` here will look weird when we get rid of it eventually 🙈 but it's not a big deal anyway

Reply:
it was setup to use same as Travis, but later I decided to use own config for GHA, so this is just leftover.

I'll probably use `zftest` (like db name), rather `root`, should not test with "root" user.